src/main/resources/static/styles.css

```diff
@@ -0,0 +1,3 @@
+body {
+color: red;
```

Comment:
is this for all body tags?

Reply:
Sorry that is when I first started, I have commented out my css file

Reply:
No need to apologize! All good!